src/sync/treiber_stack.rs

```diff
@@ -77,6 +77,12 @@ impl<T> TreiberStack<T> {
     }
 }
 
+impl<T> Default for TreiberStack<T> {
+    fn default() -> Self {
+        TreiberStack { head: Atomic::null() }
```

Comment:
Oh okay, didn't really contribute to the Rust compiler / libstd yet. If you still want me to change it, just leave another comment.